packages/x-charts/src/ChartsYAxis/ChartsYAxis.tsx

```diff
@@ -128,6 +128,7 @@ function ChartsYAxis(inProps: ChartsYAxisProps) {
           {...slotProps?.axisLine}
         />
       )}
+
```

Comment:
I don't understand why this is added on `yarn proptypes`, not on `yarn eslint`.
If anyone has any clues - let me know 🙂

Reply:
Arf

Reply:
I think that it is because we are running `prettier` here:
https://github.com/mui/mui-x/blob/6491eb941641019924d89a964addb3afe8e2c7f1/docs/scripts/generateProptypes.ts#L114

Reply:
Makes sense, thanks @LukasTy!